src/server/api/organization.js

```diff
@@ -28,7 +28,19 @@ export const resolvers = {
     },
     people: async (organization, { role, campaignId }, { user }) => {
       await accessRequired(user, organization.id, 'SUPERVOLUNTEER')
-      return buildUserOrganizationQuery(r.knex.select('user.*'), organization.id, role, campaignId)
+      let users = await buildUserOrganizationQuery(r.knex.select('user.*'), organization.id, role, campaignId)
```

Comment:
I think we should separate this from the captilize PR.  If we were going to do this, I think it would be better to change sorting in `buildUserOrganizationQuery` and make it `orderBy('first_name DESC')`  otherwise it could mess up pagination (if/when we have it)

Reply:
++